ui/apps/platform/src/Components/PatternFly/DayPickerDropdown.tsx

```diff
@@ -12,7 +12,22 @@ export type DayPickerDropdownProps = {
     handleSelect: (id, selection) => void;
     isEditable?: boolean;
     isRequired?: boolean;
-    intervalType: IntervalType;
+    intervalType: IntervalType | null;
+};
+
+export const daysOfWeekMap: Record<string, string> = {
+    '0': 'Sunday',
+    '1': 'Monday',
+    '2': 'Tuesday',
+    '3': 'Wednesday',
+    '4': 'Thursday',
+    '5': 'Friday',
+    '6': 'Saturday',
+};
+
+export const daysOfMonthMap: Record<string, string> = {
+    '1': 'The first of the month',
+    '15': 'The middle of the month',
 };
```

Comment:
Nit: You might be able to remove the explicit type annotations and declare these as `as const`, which will give you an exact immutable mapping of day numbers to Sun-Sat without the extra types allowed by `Record<string, string>`.